src/lib.rs

```diff
@@ -5671,11 +5671,15 @@ pub unsafe trait ByteSliceMut: ByteSlice + DerefMut {}
 ///
 /// # Safety
 ///
-/// Unsafe code may depend for its soundness on the assumption that `split_at`
-/// is implemented correctly. In particular, given `B: SplitByteSlice` and `b:
-/// B`, if `b.deref()` returns a byte slice with address `addr` and length
-/// `len`, then if `split <= len`, `b.split_at(split)` will return `(first,
-/// second)` such that:
+/// Unsafe code may depend for its soundness on the assumption that `split_at`,
+/// `try_split_at` and `split_at_unchecked` are implemented correctly. In
```

Comment:
Remove `try_split_at` now that it's no longer a method.

Reply:
Done.